python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -158,3 +158,61 @@ def asset_wipe_command(key, **cli_args):
             click.echo("Removed asset indexes from event logs")
         else:
             click.echo("Exiting without removing asset indexes")
+
+
+@asset_cli.command(name="wipe-cache")
+@click.argument("key", nargs=-1)
+@click.option("--all", is_flag=True, help="Wipe partitions status cache of all asset keys")
+@click.option("--noprompt", is_flag=True)
+def asset_wipe_cache_command(key, **cli_args):
+    r"""Clears the asset partition status cache.
+
+    Warning: Cannot be undone.
```

Comment:
I don't think we need to warn about this, because the consequence is just that queries will take a little longer.